### PR TITLE
Bug 1092966 - FM Radio app to follow text selection pattern

### PR DESCRIPTION
**Pull request by justindarc**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[justindarc Justin D'Arcangelo (justindarc)](https://github.com/justindarc) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=463f1ca2ca57)

**Comment by try-server-hook** [comment]
[justindarc Justin D'Arcangelo (justindarc)](https://github.com/justindarc) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a278cfd5e923)

File: apps/fm/style/fm.css [hunk 1]
```diff
@@ -8,6 +8,13 @@ html, body {
   font-size: 10px;
 }
 
+/**
+ * 1. No content inside the app
+ *    should be user-selectable.
+ */
+* {
+  -moz-user-select: none; /* 1 */
```
Comment (PinZhang) [inline apps/fm/style/fm.css]: Just curious, what does the comment, i.e. `/* 1 */`, mean?

Comment (justindarc) [inline apps/fm/style/fm.css]: The `1` is a reference to the above comment that says `No content inside the app should be use-selectable`. This is a useful way to comment CSS hacks for conditions like this so its understandable to someone viewing the code in the future as to why the property is needed. If there were several CSS declarations that needed explanation, they could subsequently be marked with a `2`, `3`, etc.

Comment (PinZhang) [inline apps/fm/style/fm.css]: OK, thanks.

**Pull request merged**